helper/activedirectory/client_test.go

```diff
@@ -55,6 +54,7 @@ func TestSearch(t *testing.T) {
 
 	result, _ = entry.GetJoined(FieldRegistry.PrimaryGroupID)
 	if result != "513" {
+		t.Error(err)
 		t.FailNow()
```

Comment:
Don't need the `t.FailNow()` in here any more.

Reply:
Thanks! Good catch!